submit.py

```diff
@@ -143,11 +148,15 @@
 print(f"Storing the report at {name}.npz...")
 
 np.savez(
-    f"{name}.npz",
+    f"result/{name}.npz",
```

Comment:
Saving as csv would make more sense with columns model_id, wandb_url, test accuracy, test loss

Reply:
iirc, you already implemented changes to this in your commit, so I'll not update this part.

Reply:
apparently it was a miscommunication, so I'll fix this